index.js

```diff
@@ -1,5 +1,74 @@
 'use strict';
 
+var colorConvert = require('color-convert');
+var supportsColor = require('supports-color');
+
+supportsColor = supportsColor || {};
+supportsColor.level = supportsColor.level || 0;
+
+colorConvert.rgb.rgb = function rgb (rgb) {
+	return rgb;
+};
+
+function getConverter (color) {
+	var has = function has (name) {
+		return color.hasOwnProperty(name);
+	}
+	switch (true) {
```

Comment:
I was trying to figure out a way to build these up dynamically and remove these horrid check to begin with, though properties aren't sorted anyway. This may be a [complete non-issue](https://github.com/chalk/chalk/issues/73#issuecomment-124826052)